code/tests/checkpoint_example.cpp

```diff
@@ -221,8 +230,10 @@ class CheckpointExample : public BaseTestCase {
 
     if (bytes_read != strlen(TEST_TEXT)) {
       test_res->SetError(DataTestResult::kFileDataCorrupted);
+      test_res->error_description = check_file + ": read data size wrong";
```

Comment:
Lets print what the size should be, and what size is reported?

Reply:
done